core/lens_errors.ml

```diff
@@ -0,0 +1,137 @@
+open Lens
+
+let unpack r ~die ~fmt =
+  match r with Result.Ok t -> t | Result.Error e -> fmt e |> die
```

Comment:
Yes, I'd also prefer the convention of new line per case.  Otherwise it is hard to tell what happens where.  If this is a pattern that recurs often enough that it'd be a pain to have it across multiple lines then it should be packed up as a helper function - though it looks like this is already the case.

Reply:
This is an ocamlformat default. I'm not the biggest fan of it, but I've somewhat come to terms with it in the sense that it will only do that if it is below 80 columns and it is for a very short segment of code. I could possibly try to tweak the settings.

Reply:
I see, thanks for clarifying. I don't think OCamlformat makes the right choice in this case. If you could tweak the settings that'd be great.

Reply:
I've adjusted this and introduced a `.ocamlformat` file with the correct settings.